client/templates/home/home.js

```diff
@@ -62,11 +62,7 @@ Template.gameList.helpers({
 });
 
 Template.gameList.events({
-    'click .button-small': function(e) {
-        Meteor.call('joinGame', $(e.currentTarget).attr('id'), function(error, results) {
-            if (error) {
-                console.log(error);
-            }
-        });
+    'click #joinGame': function(e) {
+        Router.go("/upload/" + $(e.target).attr("gameId"));
```

Comment:
joinGame is also called in the data function for the upload route. https://github.com/brianchitester/facequalm/blob/dev/lib/routes.js#L31
now i think it's the only place joinGame is called.